src/CountryPicker.style.js

```diff
@@ -69,6 +68,9 @@ export default StyleSheet.create({
   countryName: {
     fontSize: getHeightPercent(2.2)
   },
+  disabledCountryName: {
```

Comment:
I see no difference with countryName style. Should it be so?

Reply:
yes, we added different style in UICountryPicker, but key with default props is necessary